agent/util-scripts/pbench-move-results

```diff
@@ -172,120 +186,84 @@ for dir in `/bin/ls -ort -d */ | awk '{print $8}' | grep -v "^tools-" | grep -v
         let failures=failures+1
         continue
     fi
 
-    # Perform the actual copy
-    # if a prefix is provided, copy it to the other side - maybe this should be part of the tarball?
-    prefixfile=""
-    if [ ! -z "$prefix" ] ;then
-        prefixfile=$pbench_tmp/prefix.$pbench_run_name
-        echo "$prefix" > $prefixfile
-    fi
-
-    ssh -i $pbench_bin/id_rsa $ssh_opts $results_repo "mkdir -p $results_full_path"
-
-    # check if there is a name collision and resolve it
-    typeset -i i=1
-    rtarball=$tarball
-    rprefixfile=$prefixfile
-    while (( 1 )) ; do
-        ssh -i $pbench_bin/id_rsa $ssh_opts $results_repo "test -f $results_full_path/$rtarball"
-        if [ $? -eq 0 ] ;then
-            # collision - warn the first time around
-            if [ $i -eq 1 ] ;then
-                log "WARNING: name collision - $results_repo:$results_full_path/$rtarball exists"
-            fi
-        else
-            # collision resolution found
-            if [ "$tarball" != "$rtarball" ] ;then
-                mv $tarball $rtarball
-                mv $tarball.md5 $rtarball.md5
-                tarball=$rtarball
-                if [ ! -z "$prefixfile" ]; then
-                    mv $prefixfile $rprefixfile
-                    prefixfile=$rprefixfile
-                fi
-            fi
-            break
-        fi
-        rtarball="DUPLICATE__NAME.$i.$pbench_run_name.tar.xz"
-        if [ ! -z "$prefixfile" ] ;then
-            rprefixfile="$prefixfile.$i"
-        fi
-        i=$i+1
-    done
-
-    # FIXME: don't assume final path contains /incoming in it
-    if [[ $i -eq 1 ]]; then
-        if [[ ! -z "$prefix" ]]; then
-            debug_log "copying $tarball to http://$results_webserver/results/$hostname/$prefix/..."
-        else
-            debug_log "copying $tarball to http://$results_webserver/results/$hostname/..."
-        fi
-    else
-        debug_log "archiving $tarball to $results_webserver, but not being made available via the web"
-    fi
-
     # finally do the copy
-    scp $scp_opts -i $pbench_bin/id_rsa $ssh_opts ./$tarball ./$tarball.md5 $prefixfile $results_repo:$results_full_path
+    scp -r $scp_opts -i $pbench_bin/id_rsa $ssh_opts $tmp/$controller $results_repo:$results_path_prefix
     if [ $? -ne 0 ]; then
-        error_log "ERROR: unable to copy results tarball, $tarball, to $results_repo:$results_full_path"
-        rm -f $tarball $tarball.md5
+        error_log "ERROR: unable to copy results tarball, $tarball, to $results_repo:$results_path_prefix"
+        rm -f $tarball $tarballmd5
         let failures=failures+1
         continue
     fi
 
-    # clean up the prefix file (if present)
-    if [ ! -z "$prefixfile" -a -f "$prefixfile" ] ;then
-        rm -f $prefixfile
-    fi
-
     # Verify the bits copied are good
-    ssh -i $pbench_bin/id_rsa $ssh_opts $results_repo "cd $results_full_path; md5sum --check $pbench_run_name.tar.xz.md5"
+    md5name=$pbench_run_name.tar.xz.md5
```

Comment:
Could this be `$(basename $tarballmd5)`?

Reply:
Changed it to `$(basename $tarball).md5` instead.